src/Temporalio.Extensions.Hosting/README.md

```diff
@@ -0,0 +1,93 @@
+# Hosting and Dependency Injection Support
+
+This extension adds support for worker
+[Generic Host](https://learn.microsoft.com/en-us/dotnet/core/extensions/generic-host)s and activity
+[Dependency Injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection) to the
+[Temporal .NET SDK](https://github.com/temporalio/sdk-dotnet).
+
+⚠️ UNDER ACTIVE DEVELOPMENT
+
+This SDK is under active development and has not released a stable version yet. APIs may change in incompatible ways
+until the SDK is marked stable.
+
+## Quick Start
+
+Add the `Temporalio.Extensions.Hosting` package from [NuGet](https://www.nuget.org/packages/Temporalio). For example,
```

Comment:
Is this the right link?

Reply:
Ah, no, will update (though this one doesn't exist there yet)

Reply:
Fixed (and fixed in OTel one where it was wrong too)